integration_tests/Dockerfile

```diff
@@ -3,9 +3,7 @@ FROM wazopbx/xivo-purge-db
 ADD . /usr/src/xivo-purge-db
 WORKDIR /usr/src/xivo-purge-db
 
-RUN pip install -Ur requirements.txt
-RUN pip install -Ur integration_tests/requirements.txt
-RUN python setup.py develop
+RUN pip install -r integration_tests/requirements.txt
```

Comment:
You need to reinstall/update dependencies, since the xivo-purge-db image is only make on new code, the tests must have the latest code.
Also the `python setup.py develop` is useful to avoid rebuild image at each modification

Reply:
Since Dockerfile use ADD both images will be always rebuild, or do I miss something ?